### ISSUE TITLE
Default to Wayland

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/distribution#2784]
It is hight time to switch to Wayland as a default display server for a sddm and Plasma-workspace.
Keep only Xwayland for compatibility.

**Comment by niansa** [comment]
Last time I tried wayland it was pretty much unusable, on plasma at least. Lots of features like window previews didn't work.

**Comment by tpgxyz** [comment]
@niansa thanks for the help, things has changed a lot over the time

**Comment by rugyada** [comment]
Disagree. Sorry.

**Comment by tpgxyz** [comment]
@rugyada disagree to what ? To stay with an obsolete X11 ?

**Comment by rugyada** [comment]
> @rugyada disagree to what ? 

@tpgxyz 
to wayland *as default*.
Just my own humble opinion: not yet, as default.

IMO would be better to give the user the choice, since afaik everyone can start with x11 and easily change default to wayland, not everyone can do the opposite.
Thanks for considering.

**Comment by saber716rus** [comment]
> @rugyada disagree to what ? To stay with an obsolete X11 ? Can you please elaborate a little bit more what is wrong? i know that decisions here are made on "single words" rather than on deep dive.

In rosa Linux, they switched to wayland, and Wayland still does not work well with plasma 5. More precisely, the screen blocker sometimes goes to a black screen, you have to restart it. So, we need a smooth transition period until all the wayland defects are eliminated.

**Comment by fleurc** [comment]
I don't believe all the wayland defects can be eliminated, but i do think that if nobody uses it or tests it then it just gets worse with time. The best that can be done is wait for Plasma 6 and then switch to that on wayland.

**Comment by rugyada** [comment]
> but i do think that if nobody uses it or tests it then it just gets worse with time.

Not a good reason to make it as default.
FWIW in Plasma6 it looks to behave a bit better, also because meanwhile time passed and one can only assume as many bugs as possible got fixed. Still for many users it's not the best choice.

**Comment by AngryPenguinPL** [comment]
Maybe we can set it for Lx 6.1 or 7 release?
Gnome iso use Wayland by default from years...

**Comment by davidebeatrici** [comment]
I've been using Wayland for a few months now and there are two serious issues:

1. Scaling is severely broken for GTK programs such as virt-manager.
2. The clipboard doesn't work correctly when copying on/from X11 windows. More specifically, the first time goes well, but then whatever content in Wayland's clipboard ends up overwriting the content that is freshly copied in an X11 window when it loses focus.

**Comment by berolinux** [comment]
I've been using it for some time now, no serious problems here anymore. Chances are it still doesn't work well  inside VBox though.

**Comment by ZeroAbility** [comment]
The project does not have a good front end to get support for an average user, let alone for developers. Through some of my research to provide answers to our users regarding things like multi head configurations and nvidia, there are a lot of automated responses from their Bug Bot blaming everything but wayland for problems that are clearly not reproduced in X11. I don't think that would be an ideal default user experience. We have a hard enough time trying to compete with the *Fear of Missing Out* effect generated by the newness and eye candy of wayland when it comes to explaining that it's not even Alpha quality after more than a decade of development.
